src/Altinn.Apps/AppTemplates/AspNet/App.IntegrationTests/Data/apps/dibk/nabovarsel/config/process/process.bpmn

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<bpmn:definitions id="Altinn_SingleDataTask_Process_Definition"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns:altinn="http://altinn.no"
+  xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL"
+  xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI"
+  xmlns:dc="http://www.omg.org/spec/DD/20100524/DC"
+  xmlns:di="http://www.omg.org/spec/DD/20100524/DI"
+  targetNamespace="http://bpmn.io/schema/bpmn" >
+  <bpmn:process id="SingleDataTask" isExecutable="false">
```

Comment:
SingleDataTask might be confusing considering the two data tasks.

Reply:
agree